app/lib/functions.php

```diff
@@ -5,4 +5,16 @@
 		function is_user_logged_in() {
 			return User::is_user_logged_in();
 		}
+	}
+
+	if ( ! function_exists( 'base_url' ) ) {
+		function base_url() {
+			return BASE_URL;
+		}
+	}
+
+	if ( ! function_exists( 'get_table' ) ) {
+		function get_table() {
+			return TABLE_PREFIX;
```

Comment:
`get_table()` should return the table name, not the prefix. This function should have a param, then add the prefix with the param and return as the table name.